libs/eslint-plugin-vx/src/rules/no_jest_to_be.ts

```diff
@@ -0,0 +1,45 @@
+import { AST_NODE_TYPES, TSESLint, TSESTree } from '@typescript-eslint/utils';
+import { createRule } from '../util';
+
+const rule: TSESLint.RuleModule<'noJestToBe', readonly unknown[]> = createRule({
+  name: 'no-jest-to-be',
+  meta: {
+    docs: {
+      description: 'Use `toEqual` rather than `toBe` in jest assertions.',
+      recommended: 'error',
+      suggestion: false,
+      requiresTypeChecking: false,
+    },
+    fixable: 'code',
+    messages: {
+      noJestToBe: 'Use `toEqual` rather than `toBe` in jest assertions.',
+    },
+    schema: [],
+    type: 'problem',
+  },
+  defaultOptions: [],
+
+  create(context) {
+    return {
+      MemberExpression(node: TSESTree.MemberExpression) {
+        if (
```

Comment:
Ensure we don't match things like this: `expect(…)[toBe]()`, though such a thing will surely never be written in vxsuite.
```suggestion
        if (
          !node.computed &&
```